### ISSUE TITLE
stress.yaml break with Ansible 2.x

### ISSUE THREAD

**Issue by tzach** [scylladb/cassandra-test-and-deploy#84]
```
TASK [debug] *******************************************************************
fatal: [54.200.12.11]: FAILED! => {"failed": true, "msg": "'ansible.vars.hostvars.HostVars object' has no attribute u'groups.DB | intersect (groups.us_west_2)'"}
```

**Comment by tzach** [comment]
Maybe the cause is
https://github.com/ansible/ansible/issues/14067

**Comment by tzach** [comment]
The following extra debug in stress.yaml

```
 - debug: msg="{{ hostvars[item].ec2_private_ip_address }}"
      with_items: groups.{{region_group}}
```

cause

```
TASK [debug] *******************************************************************
task path: /home/fedora/cassandra-test-and-deploy/stress.yaml:28
fatal: [54.186.77.227]: FAILED! => {"failed": true, "msg": "'ansible.vars.hostvars.HostVars object' has no attribute u'groups.us_west_2'"}
```

Look like the groups defined in `inventories/ec2/hosts` are not loaded in ansible 2.0

Maybe related to https://github.com/ansible/ansible/issues/14067 ?

**Comment by tzach** [comment]
Solved by #118
